lib/experimental/Information/ModuleAvatar/index.tsx

```diff
@@ -0,0 +1,103 @@
+import * as Icons from "@/icons"
+import { cva, type VariantProps } from "class-variance-authority"
+
+const moduleIconNames = [
+  "Calendar",
+  "ClockIn",
+  "Documents",
+  "Engagement",
+  "Finance",
+  "Goals",
+  "Home",
+  "Inbox",
+  "Kudos",
+  "MyDocuments",
+  "Organization",
+  "Payroll",
+  "Performance",
+  "Profile",
+  "Projects",
+  "Recruitment",
+  "Reports",
+  "Shifts",
+  "Software",
+  "Spaces",
+  "Spending",
+  "Tasks",
+  "TimeOff",
+  "TimeTracking",
+  "Trainings",
+  "Workflows",
+] as const
+
+export const iconComponents = moduleIconNames.reduce(
+  (acc, name) => {
+    const value = Icons[`Module${name}` as keyof typeof Icons]
+    if (value) acc[name] = value
+    return acc
+  },
+  {} as Record<
+    (typeof moduleIconNames)[number],
+    React.ComponentType<React.SVGProps<SVGSVGElement>>
+  >
+)
+
+export type IconType = keyof typeof iconComponents
+
+const moduleAvatarVariants = cva("relative flex items-center justify-center", {
+  variants: {
+    size: {
+      sm: "h-5 w-5",
```

Comment:
we need to set `shrink-0` to the base style, otherwise it will be squished in narrow flex containers